Tasks/SqlAzureDacpacDeployment/DeploySqlAzure.ps1

```diff
@@ -39,6 +39,30 @@ Import-Module "Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs"
 # Load all dependent files for execution
 Import-Module ./Utility.ps1 -Force
 
```

Comment:
Name it ThrowIfMultipleFilesOrNoFilePresent. You donot have to return any thing from function